packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code.vue

```diff
@@ -54,40 +65,41 @@
 					</tera-asset-block>
 				</template>
 				<template #footer>
-					<span style="margin-right: auto"
-						><label>Model framework:</label
+					<span
+						><label>Model framework</label
 						><Dropdown
-							class="w-full md:w-14rem"
+							size="small"
 							v-model="clonedState.modelFramework"
 							:options="modelFrameworks"
 							@change="setKernelContext"
 					/></span>
-					<Button
-						:disabled="isProcessing || allCodeBlocks.length === 0"
-						label="Run"
-						icon="pi pi-play"
-						severity="secondary"
-						outlined
-						@click="handleCode"
-					/>
+					<span style="margin-right: auto">
```

Comment:
If you have to do it, use the primevue classes;
https://primeflex.org/margin#classes

```suggestion
					<span class="mr-auto">
```

Reply:
😈